README.md

```diff
@@ -9,106 +9,236 @@
-                return res(ctx.status(403))
-            }),
-        }
-    }
+  msw: {
+   handlers: {
+    auth: rest.get('/login', (req, res, ctx) => {
+      return res(ctx.status(403))
+    }),
+   }
+  }
 }
 
+```
+
+What if you want to disable global handlers? All you have to do is set them as null and they will be ignored for your story:
+```js
 // This story will disable the auth handlers from preview.js
 NoAuthBehavior.parameters = {
-    msw: {
-        handlers: {
-            auth: null,
-            others: [
-                rest.get('/numbers', (req, res, ctx) => {
-                    return res(ctx.json([1, 2, 3]))
-                }),
-                rest.get('/strings', (req, res, ctx) => {
-                    return res(ctx.json(['a', 'b', 'c']))
-                }),
-            ],
-        }
-    }
+  msw: {
+   handlers: {
+    auth: null,
+    others: [
+      rest.get('/numbers', (req, res, ctx) => {
+       return res(ctx.json([1, 2, 3]))
+      }),
+      rest.get('/strings', (req, res, ctx) => {
+       return res(ctx.json(['a', 'b', 'c']))
+      }),
+    ],
+   }
+  }
 }
 ```
 
-[Full Documentation](https://msw-sb.vercel.app/)
+#### Configuring MSW
+
+`msw-storybook-addon` starts MSW with default configuration. If you want to configure it, you can pass options to the `initialize` function. They are the [StartOptions](https://mswjs.io/docs/api/setup-worker/start) from setupWorker.
+
+A common example is to configure the `onUnhandledRequest` behavior, as MSW logs a warning in case there are requests which were not handled.
+
+If you want MSW to bypass unhandled requests and not do anything:
+```js
+// preview.js
+import { initialize } from 'msw-storybook-addon';
+
+initialize({
+  onUnhandledRequest: 'bypass'
+})
+```
+
+If you want to warn a helpful message in case stories make requests that should be handled but are not:
+```js
+// preview.js
+import { initialize } from 'msw-storybook-addon';
+
+initialize({
+  onUnhandledRequest: ({ method, url }) => {
+    if (url.pathname.startsWith('/my-specific-api-path')) {
+      console.error(`Unhandled ${method} request to ${url}.
+
+        This exception has been only logged in the console, however, it's strongly recommended to resolve this error as you don't want unmocked data in Storybook stories.
+
+        If you wish to mock an error response, please refer to this guide: https://mswjs.io/docs/recipes/mocking-error-responses
+      `)
+    }
+  },
+})
+```
+
+#### Using this addon in node
+
+Storybook provides libraries such as [@storybook/testing-react](https://github.com/storybookjs/testing-react), which allows you to reuse stories in your jest tests.
+If you are using stories in your tests, `msw-storybook-addon` will work as intended, but instead of using a service worker implementation, it will use a server implementation. `msw-storybook-addon` provides access to the server instance, in case you'd like to do something with it:
```

Comment:
I wonder if this isn't internal details information that the end consumer may not need to know. 🤔

Reply:
Perhaps we can mention what the consumer can do without diving into internal specifics. For example:

> When using this add-on in Node.js you can get the reference to the worker instance to use its [API](#).

 I really think we should rename the `getWorker` function to make sense in Node.js as well. A couple of suggestions:

- `getInstance`
- `getMockInstance`

Reply:
I think for now I'll just remove this section altogether. We can work on the function name on another PR!